openslides/core/static/js/core/base.js

```diff
@@ -476,6 +476,12 @@ angular.module('OpenSlidesApp.core', [
 ])
 
 // Template hooks
+// 2 possible uses:
+// - { Id: 'myHookId', template: '<button>click me</button>' }
+// - { Id: 'myHookId', templateUrl: '/static/templates/plugin_name/my-hook.html' }
```

Comment:
I searched a bit but I didn't find any good solution. This implementation is very small and on point whats going on, so I would rather keep it.